dotCMS/src/main/java/com/dotcms/rendering/velocity/servlet/VelocityServlet.java

```diff
@@ -70,24 +69,37 @@ public static PageMode processPageMode (final User user, final HttpServletReques
         }
 
         if (LoginMode.UNKNOWN == loginMode) {
-
-            return user.isFrontendUser()
-                    ? PageMode.setPageMode(request, PageMode.LIVE)
-                    :  useNavigateMode(request, loginMode)
-                    ? PageMode.setPageMode(request, PageMode.NAVIGATE_EDIT_MODE)
-                    :  PageMode.setPageMode(request, PageMode.LIVE);
+            return determinePageMode(request, user, LoginMode.UNKNOWN);
         }
 
         if ( LoginMode.FE == loginMode) {
-            return PageMode.setPageMode(request, PageMode.LIVE);
+            return PageMode.setPageMode(request, PageMode.LIVE, false);
```

Comment:
It would be better to set this to a cons to be more descriptive such as NO_RESPECT_FE_ROLES or so